charts/k8ez/templates/extra-deployments.yaml

```diff
@@ -1,47 +1,56 @@
 {{- if .Values.extraDeployment.enabled -}}
 {{- range .Values.extraDeployment.deployments }}
+{{- $deploymentName := (required "Each entry in extraDeployment.deployments needs a name" .name) }}
+{{- $dictOfBasicLabels := include "chart.basicLabels" $ | fromYaml }}
+{{- $dictOfBasicSelectorLabels := include "chart.basicSelectorLabels" $ | fromYaml }}
```

Comment:
what are we doing to make sure cron jobs, jobs, statefulsets, and deployments don't have overlapping selector labels?

Reply:
I think just statefulsets and deployments will run into this potential issue. Jobs/cronjobs create a dynamic label if I remember correctly.

Reply:
Basic selector labels are the app and release.
Basic labels are the app, release, chart, heritage, and version.
I didn't remove anything that differentiated between apps. I only replaced groups of common labels.
You're right, though, that we should add to the statefulset and extra deployments' labels. They need a distinguishing label.